frontend/scss/photographerListView.scss

```diff
@@ -1,9 +1,30 @@
 .one-photographer{
     flex-direction: row;
     justify-content: space-evenly;
+    width: 13.7%;
 
 }
 
+.photographers-container{
+    padding: 1px;
+}
+
 .child{
+    margin-left: 10px;
```

Comment:
Looks good. Just FYI -- one way you can do this in one line is:
`margin: VERTICAL HORIZONTAL`

so in this case
`margin: 20px 10px`

see https://developer.mozilla.org/en-US/docs/Web/CSS/margin for docs on this.